components/brave_sync/ui/components/enabledContent.tsx

```diff
@@ -111,6 +111,10 @@ export default class SyncEnabledContent extends React.PureComponent<Props, State
     })
   }
 
+  onUserNoticedError = () => {
+    this.props.actions.resetSyncSetupError()
```

Comment:
SYNC_RESET_SETUP_ERROR, this naming is confusing. what exactly is it related to reset?

Reply:
this dispatches a message cleaning up the error message (store receives `undefined`)

Reply:
per discussion in DM, please do a follow-up for better naming of this every where.